target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-x80-5g.dts

```diff
@@ -0,0 +1,488 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Robert Marko <robimarko@gmail.com> */
+/* altered by Jonathan Brophy <professor_jonny@hotmail.com> */
+/* to suit the tplink_x80 target*/
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "tplink deco-x80-5g";
+	compatible = "tplink,deco-x80-5g", "qcom,ipq8074";
+
+	aliases {
+		 serial0 = &blsp1_uart5;
+ 		 ethernet0 = &dp5; // Aliases to capture wan mac from bootloader
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+	modem {
+		compatible = "gpio-export";
+		pinctrl-0 = <&modem_pins>;
+		pinctrl-names = "default";
+
+		w_disable {
+			gpio-export,name = "w_disable";
+			gpio-export,output = <1>;
+			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>; // wwan disable output
+		};
+
+		modem-reset {
+			gpio-export,name = "modem_reset";
+			gpio-export,output = <0>;
+			gpios = <&tlmm 29 GPIO_ACTIVE_HIGH>; // reset modem output
+		};
+	};
+
+	antennas {
+		compatible = "gpio-export";
+		pinctrl-0 = <&antenna_pins>;
+		pinctrl-names = "default";
+
+		ext_antenna_en {
+			gpio-export,name = "ext_antenna_en";
+			gpio-export,output = <0>;
+			gpios = <&tlmm 56 GPIO_ACTIVE_HIGH>; // select external sma antennas for WWan output
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&button_pins>;
+		pinctrl-names = "default";
+
+		reset_button {
+			label = "reset";
+			gpios = <&tlmm 27 GPIO_ACTIVE_LOW>; //reset button output
+			linux,code = <KEY_RESTART>; // Aliases for button to stitch to linux reset functioon
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = <&led_pins>;
+		pinctrl-names = "default";
+		// note the LED'd defined here are not individual they are a single combined RGB element
+		//to mimic the OEM default behavior green and red are switched on from boot to indicate yellow
+		//to show the device is booting behavior can be user defined in procd to override this.
+		//OEM behaviour of multi function LED:
+		//pulse Yellow: resetting/ rebooting.
+		//Solid Yellow: starting up.
+		//Pulse Blue: factory reset/ first boot ready for setup or WPS in progress.
+		//Solid Blue: setting up/ comitting settings change.
+		//Pulse Green: upgrading firmware.
+		//Solid Green: connected to 4G / WAN.
+		//Solid Purple: connected to 5G.
+		//Pulse Red: mesh failure cant find peers.
+		//Solid Red: fault
```

Comment:
Please use `/* */` instead of `//` for comments.

Reply:
Is this just for comments on their own line or does this include comments in line with code also?